### ISSUE TITLE
Fix statusBarStyle being reset to black after modal window has closed

### ISSUE THREAD

**Issue by FokkeZB** [viezel/NappDrawer#56]
Nice to have #13 fixed, but when you open a modal window and then close it, the statusBarStyle again is black. This means I still need to use the hack:

```
    // open drawer
    $.drawer.open();

    // WORKAROUND https://github.com/viezel/NappDrawer/issues/13
    if (OS_IOS) {
        Ti.UI.createWindow({
            backgroundColor: 'transparent',
            zIndex: 1000,
            touchEnabled: false
        }).open();
    }
```

Would be nice to have this fixed in a good way.  What are you doing in creating the windows for nappdrawer that https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiWindowProxy.m doesn't?

**Comment by PlutonB** [comment]
Any news on this issue? It's the same problem when using the function hideStatusBar() to hide the statusbar. As soon as a modal window has opened and close the statusbar is back. Plus I'm having trouble to avoid flickering of the statusbar when changing windows.

The optimal solution would of course be if NappDrawer could adhere to the settings for having the statusbar shown or hidden in TiApp.xml, but this is maybe not possible technically?

**Comment by hansemannn** [comment]
Fixed by #219
